data/hai/midas.txt

```diff
@@ -0,0 +1,1010 @@
+			choice
+				`	"Who are 'they'?"`
+					goto who
+				`	"I got these after a duel."`
+				`	"Where I got these bars is none of your business."`
+					goto wontsay
+				`	(Ignore him.)`
+					decline
+			`	"Ha! I made the same mistake, once, plundering them from a ship. It almost was my last."`
+			label who
+			`	He looks at you, fascinated. "You truly do not know, do you?" He stops for a few seconds, examining your face as if the truth was hidden behind your left ear. "Good! Then you may still be free. Do not make the mistake I did, or this gold will corrupt you! They will come for it! Stash it away, or sell it whilst you still can! It is not worth the fabulous prices they may offer for it!"`
+			`	He continues, finally and painfully catching his breath. "I went to the pirate worlds! To their territory! What a fool I was. Yes, it was a lot of credits, but not for me. I got plundered myself on the way there!"`
+			choice
+				`	"A good price you say? That sounds interesting."`
+				`	"You're right, it is not worth it, I will get rid of them."`
+					goto rid
+				`	"I just sold all of my bars but if they are this interesting I'll be looking for more."`
+					to display
+						not "outfit (all): Gold Bars"
+			`	His eyes open fully in despair, "NO!! You fool! You won't be able to say I did not warn you. You truly are a lost cause, perhaps the bars have already affected you! At least, make sure to have enough escorts, and watch your back.`
+			`	"At least do not keep them on your flagship, that would only attract them," he shouts over his shoulder before running away.`
+				accept
+			label rid
+			`	"Good. You do well to listen." he says, before leaving, finally calm.`
+				decline
+			label wontsay
+			`	"It is your choice to make. Just know that it is a mistake, just like it would be to take these to a pirate world. Or do anything else than getting rid of them." he says, before leaving.`
+	npc evade
+		government "Midas"
+		personality plunders disables heroic nemesis marked
+		system
+			distance 2 5
+		fleet "Marauders" 3
+	to fail
+		or
+			not "outfit: Gold Bars"
+			has "Midas: Betrayal For The Bars: offered"
+	to complete
+		never
+
+
+
+
+phrase "midas defeated"
+	word
+		`After disabling the last of the pirate ships, you wonder what could have motivated such an attack. Do you possess something valuable to them? When they entered the system, they only sent one message: "nemo me impune lacessit". It is probably best to store whatever they were interested in on a planet (using U) or to get rid of it.`
```

Comment:
```suggestion
		`After getting rid of the last of the pirate ships, you wonder what could have motivated such an attack. Do you possess something valuable to them? When they entered the system, they only sent one message: "Nemo me impune lacessit". It is probably best to store whatever they were interested in on a planet (using U) or to get rid of it.`
```
What if you didn’t destroy them?

Reply:
this phrase triggers upon disabling them, not destroying them

Reply:
But it also works if you destroy them due to the way the code currently works, yes?

Reply:
if you instantly kill them yeah ig
I think I applied the changes anyway